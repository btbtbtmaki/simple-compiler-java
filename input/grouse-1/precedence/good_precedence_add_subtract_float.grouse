main {

  print 8.6 + 1.9 - 1.2 nl;
  print 9.2 + 9.1 - 3.4 nl;
  print 3.3 + 5.0 - 2.9 nl;
  print 4.9 + 9.5 - 1.2 nl;
  print 5.3 + 8.9 - 6.0 nl;
  print 5.8 + 1.7 - 3.6 nl;
  print 1.5 + 9.8 - 3.3 nl;
  print 2.4 + 2.0 - 9.0 nl;
  print 7.3 + 5.1 - 4.6 nl;
  print 3.0 + 4.9 - 3.2 nl;
  print 1.7 + 7.6 - 7.8 nl;
  print 8.0 + 7.6 - 6.4 nl;

  print 5.5 - 2.6 + 8.9 nl;
  print 1.4 - 1.1 + 7.5 nl;
  print 3.3 - 8.4 + 4.4 nl;
  print 7.9 - 4.7 + 7.6 nl;
  print 6.1 - 6.4 + 3.1 nl;
  print 7.8 - 2.6 + 4.7 nl;
  print 8.2 - 9.6 + 3.2 nl;
  print 7.4 - 9.1 + 6.0 nl;
  print 9.8 - 4.5 + 9.9 nl;
  print 9.9 - 2.5 + 3.3 nl;
  print 1.1 - 4.0 + 6.8 nl;
  print 6.9 - 8.6 + 1.9 nl;

  print 1.2 + 5.0 - 9.4 + 1.2 nl;
  print 4.3 + 6.6 - 9.0 + 4.9 nl;
  print 5.6 + 5.4 - 2.5 + 9.7 nl;
  print 6.9 + 5.9 - 9.4 + 8.1 nl;
  print 2.5 + 2.5 - 9.3 + 5.9 nl;
  print 1.2 + 2.3 - 8.4 + 2.9 nl;
  print 1.2 + 6.8 - 5.7 + 3.1 nl;
  print 3.2 + 4.4 - 6.3 + 5.3 nl;
  print 1.5 + 9.3 - 4.3 + 9.9 nl;
  print 4.1 + 2.2 - 1.2 + 2.1 nl;
  print 5.0 + 8.3 - 9.8 + 8.3 nl;
  print 9.3 + 6.9 - 7.2 + 5.4 nl;

  print 1.1 - 2.6 + 2.0 - 8.0 nl;
  print 4.4 - 8.8 + 2.7 - 9.5 nl;
  print 9.8 - 7.0 + 7.5 - 6.7 nl;
  print 7.8 - 5.5 + 9.5 - 1.9 nl;
  print 2.4 - 5.1 + 8.1 - 8.4 nl;
  print 3.8 - 5.1 + 4.2 - 6.0 nl;
  print 7.2 - 8.8 + 8.0 - 4.7 nl;
  print 7.0 - 7.0 + 9.6 - 6.8 nl;
  print 6.7 - 1.4 + 6.1 - 6.7 nl;
  print 8.0 - 6.3 + 8.5 - 7.8 nl;
  print 2.3 - 1.9 + 4.8 - 4.3 nl;
  print 6.9 - 3.5 + 9.6 - 6.0 nl;

}

