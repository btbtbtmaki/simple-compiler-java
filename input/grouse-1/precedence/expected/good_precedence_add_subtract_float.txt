9.3
14.9
5.4
13.2
8.2
3.9
8
-4.6
7.8
4.7
1.5
9.2
11.8
7.8
-0.7
10.8
2.8
9.9
1.8
4.3
15.2
10.7
3.9
0.2
-2
6.8
18.2
11.5
1.6
-2
5.4
6.6
16.4
7.2
11.8
14.4
-7.5
-11.2
3.6
9.9
-3
-3.1
1.7
2.8
4.7
2.4
0.9
7
